kolibri/core/assets/src/views/AppError/index.vue

```diff
@@ -83,12 +154,19 @@
     },
     $trs: {
       defaultErrorHeader: 'Sorry! Something went wrong!',
+      // eslint-disable-next-line
       defaultErrorExitPrompt: 'Back to home',
       pageReloadPrompt: 'Refresh',
       defaultErrorMessage:
         'We care about your experience on Kolibri and are working hard to fix this issue.',
       defaultErrorResolution: 'Try refreshing this page or going back to the home page.',
       defaultErrorReportPrompt: 'Help us by reporting this error',
+      pageNotFoundHeader: 'Page not found',
+      pageNotFoundMessage: "Sorry, we can't seem to find the page you're looking for.",
```

Comment:
unnecessary trailing period. From the [writing guidelines](http://kolibribeta.learningequality.org/design/patterns/writing):

> Skip periods and unnecessary punctuation

Reply:
Actually, looks like we forgot to incorporate the latest strings: https://docs.google.com/document/d/1B7E9RrxLatDF8R9a8JG8W8tEez6KrjrwPQPbE1biKsg/edit#